src/scmdata/run.py

```diff
@@ -2200,6 +2200,48 @@ def reduce(self, func, dim=None, axis=None, **kwargs):
 
             return type(self)(data, index=index, columns=meta)
 
+    def round(self, decimals=3, inplace=False):
+        """
+        Round data to a given number of decimal places.
+
+        For values exactly halfway between rounded decimal values, NumPy rounds
+        to the nearest even value. Thus 1.5 and 2.5 round to 2.0, -0.5 and 0.5
+        round to 0.0, etc.
+
+        Parameters
+        ----------
+        decimals : int
+            Number of decimal places to round each value to.
+
+        inplace : bool
+            If True, apply the conversion inplace and return None
+
+        Returns
+        -------
+        :class:`ScmRun <scmdata.run.ScmRun>`
+            If :obj:`inplace` is not ``False``, a new :class:`ScmRun <scmdata.run.ScmRun>` instance
+            with the rounded values.
+
+        """
+        if inplace:
```

Comment:
Should we create an internal function like below

```python
def _get_ret(self, inplace):
    if inplace:
        return self

    return self.copy()
```

Reply:
Good idea. Added a `_get_target` function to do just that.

It wasn't performed in as many places as I thought it would be. There was a similar call in `filter` that used a different copy (didn't also copy the underlying data). I didn't want to touch that here without thinking through what that could mean.

Reply:
Nice, I suspected that might be the case